src/jsdom.js

```diff
@@ -29,7 +29,8 @@ function fromSource(src, options) {
             FetchExternalResources: ['script'],
             ProcessExternalResources: ['script']
         },
-        virtualConsole: jsdom.createVirtualConsole().sendTo(console)
+        virtualConsole: jsdom.createVirtualConsole().sendTo(console),
+        created: options.jsdomCreated
```

Comment:
Sure, it is a callback that is triggered on creation of the jsdom. It receives (e, window). This is how one can add on localStorage shims, etc. You can see it in action in the new test.

Uncss is currently using the "old-api" in the jsdom project, so the link is: [https://github.com/tmpvar/jsdom/blob/master/lib/old-api.js#L153](https://github.com/tmpvar/jsdom/blob/master/lib/old-api.js#L153)